Userland/Libraries/LibLine/Editor.cpp

```diff
@@ -2106,7 +2112,7 @@ Result<Vector<size_t, 2>, Editor::Error> Editor::vt_dsr()
     if (m_input_error.has_value())
         return m_input_error.value();
 
-    fputs("\033[6n", stderr);
+    fputs("\033[6n\n", stderr);
```

Comment:
Is this in here intentionally? Looks somewhat unrelated to the commit message from a distance.

Reply:
It's intentional, should be a separate commit though, will fix.